### DISCUSSION TITLE
Setup process for nextjs/ react app to integrate antmedia conference-call-application

### DISCUSSION THREAD

**Discussion by AKASH-926** [ant-media/Ant-Media-Server#6485] [Q&A]
I have followed the steps mentioned in the Ant Media documentation. Although it appears to be working, the fonts and icons are not displaying when the app is rendered.So if anyone went through this issue please guide me with the setup process.

**Comment by yashtandon113** [comment]
@AKASH-926 
May I know which documentation you followed to integrate the AntMedia conference call application?

**Reply by AKASH-926** [reply]
https://antmedia.io/docs/guides/developing-antmedia-server/circle-component-usage/

I am referring to this doc.
Please let me know how to solve this issue

Issue while loading the icons
main.4c914112.js:2 Unsafe attempt to load URL https://example.cdn-url.com/live-player/ant-media-test/static/media/sprite.8644277e6ac9fc33a1911110760887ea.svg from frame with URL https://example.domain.com/admin-v3/liveplayer?liveLessonId=2228276. Domains, protocols and ports must match.

**Comment by lastpeony** [comment]
Next.js is a server side rendering framework. 
Make sure you declare all components of circle as client side rendered. 
Check this documentation PR about next.js
https://github.com/ant-media/ant-media-documentation/pull/300/files
